tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -250,18 +250,23 @@ def _run(self):
                                         hostname not in network_ip):
                                         b_index += 1
                                         continue
+                                    sub_vol_size = (int(volumes['volume%s.brickcount' % index])) / int(
```

Comment:
Yes, it will always be a whole number. Because, Every sub volume should have same number of bricks in gluster